instrumentation/netty/netty-4.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_1/server/NettyHttpServerTracer.java

```diff
@@ -92,4 +92,10 @@ protected Integer peerPort(Channel channel) {
     }
     return null;
   }
+
+  /** This method is overridden to allow other classes in this module to call it. */
```

Comment:
Does this mean that `BaseTracer.onException` should be public? We override it in so many cases.

Reply:
Yep, I'll make it public